app/src/main/java/me/hufman/androidautoidrive/carapp/carinfo/CarDetailedInfo.kt

```diff
@@ -184,11 +198,19 @@ class CarDetailedInfo(carCapabilities: Map<String, Any?>, cdsMetrics: CDSMetrics
 			speed, heading,
 			gforces, engineRpm
 	)
+	private val drivingPerformanceFields: List<Flow<String>> = listOf(
```

Comment:
I'm planning to merge #711 soon, which will provide this screen under the name "Sport Display" and also provide the engine torque metric. I fear you'll have to merge the conflict afterwards, but just a heads up!

Reply:
OK I will have a look on this.